pkg/controller/jobframework/setup_test.go

```diff
@@ -120,8 +120,8 @@ func TestSetupControllers(t *testing.T) {
 				},
 				MapperProvider: func(*rest.Config, *http.Client) (apimeta.RESTMapper, error) {
 					gvs := make([]schema.GroupVersion, len(tc.mapperGVKs))
-					for _, gvk := range tc.mapperGVKs {
-						gvs = append(gvs, gvk.GroupVersion())
+					for i, gvk := range tc.mapperGVKs {
+						gvs[i] = gvk.GroupVersion()
 					}
```

Comment:
```suggestion
gvs := slices.Map(tc.mapperGVKs, func(gvk *schema.GroupVersionKind ) schema.GroupVersion {return gvk.GroupVersion()} )
```
from `kueue/pkg/util/slices`

Reply:
Done